docs/contracts/NFT Contract/NFT Actions/fhglobalshr.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'fhglobalshr'
+order: 35
+deploy: ['experimental']
+---
+
+# fhglobalshr
+
+Ultra configures protocol fee for first hand token purchase
+
+## Technical Behavior
+
+The required authorization is the `ultra.nft.ft`
+
+`share` must not exceed the value specified in `saleshrlmcfg` table under a scope of `0` (if exists) or 1000 (10%) otherwise
```

Comment:
share` must not exceed the value specified in the `saleshrlmcfg` table under a scope of `0` (if it exists) or 1000 (10%)